### ISSUE TITLE
Doesn't add trailing commas to imports

### ISSUE THREAD

**Issue by sloria** [asottile/add-trailing-comma#52]
I would expect this to add trailing commas to multiple imports:

```python
from foo import (
    bar,
    baz,
)
```

**Comment by asottile** [comment]
Indeed! it should probably do this, let me see if it's quick and easy

**Comment by asottile** [comment]
Looks like there isn't offset information for the names which makes this slightly more work, but it's definitely doable:

```python
from . import (
    foo,
    bar
)
```

```python
(Pdb) astpretty.pprint(node)
ImportFrom(
    lineno=1,
    col_offset=0,
    module=None,
    names=[
        alias(name='foo', asname=None),
        alias(name='bar', asname=None),
    ],
    level=1,
)
```

I can probably take the same approach that https://github.com/asottile/reorder_python_imports takes to partition these

No promises on when I'll get to this but definitely think it should be done.

While looking at this it looks like classdefs don't receive any treatment either:

```python
class C(
        Base1,
        Base2,
):
    pass
```

**Comment by sloria** [comment]
Thanks for the quick response!

**Comment by asottile** [comment]
Implemented this! Thanks for the issue -- this should show up in the next release (probably 0.7.0?)

**Comment by asottile** [comment]
Indeed, just released 0.7.0 with [this feature](https://github.com/asottile/add-trailing-comma#trailing-commas-for-from-imports)!
